gprofiler/merge.py

```diff
@@ -238,6 +239,7 @@ def concatenate_profiles(
     add_container_names: bool,
     metadata: Metadata,
     metrics: Metrics,
+    add_application_names: bool = True,
```

Comment:
I would've place it next to `add_container_names` and make it non-default, or vice versa. Just for sake of coherency.